test/msw-api/setup-server/life-cycle-events/on.test.ts

```diff
@@ -156,6 +165,23 @@ test('emits events for an unhandled request', async () => {
   )
 })
 
+test("emits errors from the user's RequestHandler", async () => {
+  const url = httpServer.http.makeUrl('/error')
+  // In NodeJS errors are not gracefully transformed into 500 responses
+  await fetch(url).catch(() => undefined)
+  const requestId = getRequestId(listener)
+
+  expect(listener).toHaveBeenNthCalledWith(
```

Comment:
I believe there's a different test that ensures that `request:start` is called. We can remove this assertion from this particular test, keeping it focused on the new event we introduce.